src/vt-tv/render/render.h

```diff
@@ -258,18 +255,12 @@ struct Render {
    * \param[in] in_selected_phase the phase selected (if max then render all)
    */
   Render(
-    std::array<std::string, 3> in_qoi_request,
-    bool in_continuous_object_qoi,
-    Info& in_info,
-    std::array<uint64_t, 3> in_grid_size,
-    double in_object_jitter,
-    std::string in_output_dir,
-    std::string in_output_file_stem,
-    double in_resolution,
-    bool in_save_meshes,
+    std::array<std::string, 3> in_qoi_request, bool in_continuous_object_qoi,
+    Info& in_info, std::array<uint64_t, 3> in_grid_size,
+    double in_object_jitter, std::string in_output_dir,
+    std::string in_output_file_stem, double in_resolution, bool in_save_meshes,
     bool in_save_pngs,
-    PhaseType in_selected_phase = std::numeric_limits<PhaseType>::max()
-  );
+    PhaseType in_selected_phase = std::numeric_limits<PhaseType>::max());
```

Comment:
I think that this was a little more readable before (with each argument on a separate line). Was this change motivated by the vt style guidelines, or was it automated with the .clang-format? (if the latter, I think we can revert to the way it was). 

What do you think @pierrepebay ?

Reply:
@cwschilly @pierrepebay this was done using clang-format to match clang format rules

Reply:
@cwschilly I agree that it is not very readable
Then we can change that clang configuration options :
`BinPackArguments: false
BinPackParameters: false
`
This puts 1 parameter or arg on a separate line if line is too long when calling 
But in that case we have to accept to diverge from vt guidelines...

Some code like the following will then also be impacted
```c++
static vtkSmartPointer<vtkScalarBarActor> createScalarBarActor_(
    vtkSmartPointer<vtkMapper> mapper, const std::string& title, double x,
    double y, uint64_t font_size = 50,
    std::set<std::variant<double, int>> values = {});
```
will become
```c++
  static vtkSmartPointer<vtkScalarBarActor> createScalarBarActor_(
    vtkSmartPointer<vtkMapper> mapper,
    const std::string& title,
    double x,
    double y,
    uint64_t font_size = 50,
    std::set<std::variant<double, int>> values = {});
```
In my opinion it will be better to do like this (so change that clang rules) and more readable. There are similar rules in other language formatting tools to make code more readable.

Reply:
@tlamonthezie That sounds good to me, but let's see what @pierrepebay @nlslatt @lifflander think

Reply:
@cwschilly I applied the new rule (`BinPackArguments: false` and `BinPackParameters: false`) in my latest commits